application-licensing-licensor/application-licensing-licensor-api/src/main/resources/templates/extensionAutoUpgraded/extensionAutoUpgraded.vm

```diff
@@ -0,0 +1,53 @@
+#template('notification/macros.vm')
+
+#includeMacros('Licenses.Code.VelocityMacros')
+
+#set ($mainIcon = "bell")
+#set ($smallIcon = "arrow_up")
+#set ($compositeEventDate = $escapetool.xml($datetool.whenIs($event.dates.get(0))))
+
+#macro (displayEventDetails $event)
+  #set ($userName = $event.user.name)
```

Comment:
Seems to be used only once below, so I'm not sure it's worth having a variable (due to the way Velocity works I'm trying to avoid creating unneeded variables which may influence some other code that uses the same name in the same namespace / context).